docs/docs/deploying-to-digitalocean-droplet.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Deploying to DigitalOcean Droplet
+---
+
+This guide walks through how to deploy and host your next Gatsby site on [DigitalOcean Droplet](https://www.digitalocean.com/products/droplets/).
+
+DigitalOcean provides the easiest cloud platform to deploy, manage, and scale applications of any size, removing infrastructure friction and providing predictability so developers and their teams can spend more time building better software.
+
+DigitalOcean's famous product droplets are scalable compute IaaS(Infrastructure as a Service) or a VPS on the cloud which has great reliability and scalability and they come along with varied price range ideal for small apps to giant enterprise-level apps that harness the profit for your business.
+
+They provide service to select from the 6 Unix based distributions and select your technology-based platform with preinstalled pre-requisites from the marketplace.
+
+A Droplet could be spin up in just 55 seconds with as low as \$5/month!
+
+## Pre-requisites
+
+1. Your Gatsby site living in a Git repository (Github/Gitlab or any git cloud).
+2. [DigitalOcean Droplet](https://www.digitalocean.com/products/droplets/) with a non-root user configured with sudo group.
+3. A custom domain name for your Gatsby site which will help configure SSL for HTTPS.
+
+## Steps to deploy your Gatsby site to DigitalOcean Droplet
```

Comment:
```suggestion
## How to deploy your Gatsby site to DigitalOcean
```
This is a Reference Guide, not a tutorial, so it should focus more generally on content than steps (also why the prerequisites are recommended as an unordered list).